accept/src/index.ts

```diff
@@ -57,10 +44,22 @@ async function run() {
     const octokit = getOctokit(ghToken)
     const [owner, component] = process.env['GITHUB_REPOSITORY'].split('/')
 
+    // Get PR
+    const {
+      data: [pull_request],
+    } = await octokit.pulls.list({
+      owner,
+      repo: component,
+      sort: 'updated',
+      head: `exivity:${branch}`,
+    })
+
     // No PR found, skip
-    if (!(await hasPR(octokit, branch, component, owner))) {
+    if (
+      !pull_request?.requested_reviewers?.some((r: any) => r.id == EXIVITY_BOT)
```

Comment:
Does this also return reviewers who already have a review submitted? Because if the bot submitted a review already, we can also skip it?

Reply:
I honestly don't know 🤷 
then again, as it should only get triggered on review_requested, such a case should never happen?

Reply:
It could. Imagine this PR.
You assign me as a reviewer. Condition is false, no test triggered.
You add `exivity-bot` as a reviewer. Condition is true, test triggered.
Scaffold test runs, succeeds, review is submitted as approved.
You assign someone else than me or bot as a reviewer, condition is now (maybe?) true and another test is triggered. But it shouldn't.

Reply:
I just called the api endpoint for this PR on postman and it gave me an empty list for requested reviewers, so I will conclude that once someone gave a review, they won't be listed as a requested reviewer anymore